### PR TITLE
feat: Use 'telefon' instead of 'mobil' in Svipper/Troms

### PR DESCRIPTION
**Pull request by tfk-kaare**
Our marketing team insist on the use of 'telefon' in norwegian instead of 'mobil'.

### CODE REVIEW

File: src/translations/screens/subscreens/TravelToken.ts [hunk 1]
```diff
@@ -495,7 +495,7 @@ export default orgSpecificTranslations(SelectTravelTokenTexts, {
             'Kan eg reise utan mobil?',
           ),
           answer: _(
-            'Du må alltid ha med deg det mobilen du har valgt å bruke billetten på.',
+            'Du må alltid ha med deg den mobilen du har valgt å bruke billetten på.',
```
Comment (tft-abel) [inline src/translations/screens/subscreens/TravelToken.ts:498]:
```suggestion
            'Du må alltid ha med deg den telefonen du har valgt å bruke billetten på.',
```

File: src/translations/screens/Profile.ts [hunk 2]
```diff
@@ -439,6 +439,18 @@ export default orgSpecificTranslations(ProfileTexts, {
               'Standard billettkategori',
             ),
           },
+          travelToken: {
+            label: _(
+              'Bruk billett på reisekort / telefon',
+              'Use ticket on reisekort / telefon',
```
Comment (tft-abel) [inline src/translations/screens/Profile.ts:445]: Should we translate "reisekort"?

```suggestion
              'Use ticket on travel card / phone',
```

File: src/translations/screens/subscreens/TravelToken.ts [hunk 3]
```diff
@@ -563,34 +563,44 @@ export default orgSpecificTranslations(SelectTravelTokenTexts, {
     travelToken: {
       header: {
         title: _(
-          'Bruk billett på reisekort / mobil',
+          'Bruk billett på reisekort / telefon',
           'Use ticket on travel card / phone',
-          'Bruk billett på reisekort / mobil',
+          'Bruk billett på reisekort / telefon',
```
Comment (tft-abel) [inline src/translations/screens/subscreens/TravelToken.ts:568]: Same as above
```suggestion
          'Bruk billett på travel card / phone',
```

File: src/translations/screens/subscreens/TravelToken.ts [hunk 4]
```diff
@@ -700,9 +710,9 @@ export default orgSpecificTranslations(SelectTravelTokenTexts, {
     },
     toggleToken: {
       title: _(
-        'Bytt mellom reisekort / mobil',
-        'Switch between travel card / phone',
-        'Byt mellom reisekort / mobil',
+        'Bytt mellom reisekort/telefon',
+        'Switch between travel card/phone',
+        'Byt mellom reisekort/telefon',
```
Comment (tft-abel) [inline src/translations/screens/subscreens/TravelToken.ts:715]: In other places, we have a space between the /

```suggestion
        'Bytt mellom reisekort / telefon',
        'Switch between travel card / phone',
        'Byt mellom reisekort / telefon',
```

File: src/translations/screens/subscreens/TravelToken.ts (hunk 1 above)
Comment (tfk-kaare) [reply src/translations/screens/subscreens/TravelToken.ts:498]: Actually, this is an outside context fix on behalf of 'fram'. It's grammar correction and they should use 'mobilen' :-) If anything, this change should be left out.

File: src/translations/screens/Profile.ts (hunk 2 above)
Comment (tfk-kaare) [reply src/translations/screens/Profile.ts]: Good catch. Fixed.

File: src/translations/screens/subscreens/TravelToken.ts (hunk 4 above)
Comment (tfk-kaare) [reply src/translations/screens/subscreens/TravelToken.ts:715]: Yeh.. This is an overwhelming issue/bug in the entire translation tree. Språkrådet says there should be no [space between single words in listing of single word alternatives](https://sprakradet.no/godt-og-korrekt-sprak/rettskriving-og-grammatikk/tegn/skrastrek/). Though, to have this typo being consistent, I'll 'fix' this

**Comment by gorandalum** [comment]
Isn't this something the marketing teams in the OMS can sit down together and agree on?

I am pretty certain that this will at some point cause problems, since there will probably be situations going forward places where we write "mobilen" while forgetting that Troms should use "telefonen".

**Comment by tfk-kaare** [comment]
I whole-heartedly agree. I see this translation as a short-term fix to please our customer base, but I think the society is overall aligned with the use of 'telefon' to mean 'mobiltelefon' or 'smarttelefon'.

So.

Let's just create a new ticket to raise this suggestion to the entire OMS?

**Comment by Aliaaen** [comment]
We should of course strive to use consistent language and wording, so using the same word everywhere and for everyone sounds nice :) However, I think that these words are pretty much interchangeable at this point, so I'm curious as to why your marketing department is so adamant about it? What is their reasoning?

**Comment by tfk-kaare** [comment]
I suspect our marketers reasons are along these lines:
1) The difference between 'telefon' to mean house phone has completely faded out, as everyone now has mobile phones. So there is no longer need to differentiate these two.
2) Even though the terms are interchangeable to mean the same, I suspect there is a shift/lean towards the use of 'telefon'
3) Consistency. Our language teacher always said that you are free to chose the word/spelling you'd like when there was options, but stick to it.

**Pull request merged**